docroot/modules/humsci/hs_config_readonly/src/EventSubscriber/ConfigReadOnlyEventSubscriber.php

```diff
@@ -49,9 +49,16 @@ class ConfigReadOnlyEventSubscriber implements EventSubscriberInterface {
    */
   protected $readOnlyFormIds = [
     'config_single_import_form',
-    'system_modules',
```

Comment:
Based on your title it doesn't seem that these subtractions were intentional. Are you sure you want to remove these?

Reply:
yeah. they are going to want to enable modules and change permissions on each site. thanks for checking

Reply:
i purposefully left the module disable since it could cause issues with the configs if they try to re-enable